client/src/components/Login/Login.tsx

```diff
@@ -85,7 +85,12 @@ class Login extends Component<Props, State> {
                         validationSchema={loginSchema}
                     >
                         {(props) => (
-                            <div className={'form-container'}>
+                            <div className={'form-container'} onKeyDown={(e) => {
+                                if(e.key === 'Enter') {
+                                    props.values.email === ''  && props.values.password === '' ?
```

Comment:
Not that big of a deal, but it would be nice to add a visual feedback when pressing enter, just for the user to know that the form was submitted. If credentials are invalid it is not obvious the form is being submitted. Same style change as on hover is just fine.